modules/card/react/stories/stories.mdx

```diff
@@ -0,0 +1,27 @@
+import {Meta, Story, Preview, Props} from '@storybook/addon-docs/blocks';
+
+import {spacing} from '../../../core/react/index';
+import README from '../README.md';
+import Card from '../index';
+
+<Meta title="Card" component={Card} />
+
+# Card
+
+<Preview>
+  <Story name="Default">
```

Comment:
@sahlhoff and I were talking about this. MDX seems to break the current belief that the order stories are loaded effects the order in the navigation - even though that advice comes from the Storybook core team. It could change later